src/pybind/mgr/dashboard/services/rbd.py

```diff
@@ -415,3 +434,43 @@ def _remove_snapshot(ioctx, img, snapshot_name, unprotect):
         pool_name, namespace, image_name = parse_image_spec(image_spec)
         return rbd_image_call(pool_name, namespace, image_name,
                               _remove_snapshot, snapshot_name, unprotect)
+
+
+class RbdMirroringService:
+
+    @classmethod
+    def enable_image(cls, image_name: str, pool_name: str, namespace: str, mode: str):
+        def _enable(ioctx, image, mode):
+            image.mirror_image_enable(mode)
+        rbd_image_call(pool_name, namespace, image_name, _enable, mode)
```

Comment:
For simple (oneliner) nested methods, why not using lambdas?
```suggestion
    @classmethod
    def enable_image(cls, image_name: str, pool_name: str, namespace: str, mode: str):
        rbd_image_call(pool_name, namespace, image_name, lambda ioctx, img, mode: image.mirror_image_enable(mode), mode)
```
I always find nested method a bit disruptive to review...